spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -54,6 +55,18 @@ The ICS 20 token denominations are represented by the form `{ics20Port}/{ics20Ch
 
 A sending chain may be acting as a source or sink zone. When a chain is sending tokens across a port and channel which are not equal to the last prefixed port and channel pair, it is acting as a source zone. When tokens are sent from a source zone, the destination port and channel will be prefixed onto the denomination (once the tokens are received) adding another hop to a tokens record. When a chain is sending tokens across a port and channel which are equal to the last prefixed port and channel pair, it is acting as a sink zone. When tokens are sent from a sink zone, the last prefixed port and channel pair on the denomination is removed (once the tokens are received), undoing the last hop in the tokens record. A more complete explanation is [present in the ibc-go implementation](https://github.com/cosmos/ibc-go/blob/457095517b7832c42ecf13571fee1e550fec02d0/modules/apps/transfer/keeper/relay.go#L18-L49).
 
+The memo is not parsed by the ICS20 application and may be used for external users or higher-level middleware and smart contracts. If the memo is intended to be used within the state machine, it must be in JSON dictionary format with the relevant information keyed for the intended user.
```

Comment:
TODO: Add Link to reserved keywords. cc: @womensrights

Reply:
the memo keys are [here](https://github.com/cosmos/chain-registry/blob/master/_memo_keys/ICS20_memo_keys.json)